lib/rubocop/cop/style/block_delimiters.rb

```diff
@@ -338,7 +338,12 @@ def braces_style?(node)
         def correction_would_break_code?(node)
           return unless node.keywords?
 
-          node.send_node.arguments? && !node.send_node.parenthesized?
+          node.send_node.arguments? && !node.send_node.parenthesized? ||
```

Comment:
I'd probably have each expression on its own line - makes the code a bit easier to read. On the other hand without parentheses some people might be confused about the precedence of the operators. I'm surprised we don't have a cop about this already (suggesting making precedence explicit).